functions.opengraph.php

```diff
@@ -94,6 +94,24 @@ function jetpack_og_tags() {
 			$tags['profile:first_name'] = get_the_author_meta( 'first_name', $author->ID );
 			$tags['profile:last_name']  = get_the_author_meta( 'last_name', $author->ID );
 		}
+	} elseif ( is_archive() ) {
+		$tags['og:type'] 		= 'website';
+		$tags['og:title']		= wp_get_document_title();
+		$tags['og:site_name']	= get_bloginfo( 'name' );
+
+		$archive = get_queried_object();
+		if ( ! empty( $archive ) ) {
+			if ( is_category() || is_tag() || is_tax() ) {
+				$tags['og:url'] 		= get_term_link( $archive->term_id, $archive->$taxonomy );
+				$tags['og:description']	= $archive->description;
```

Comment:
Here and below, is `$archive->description` always set? Should we account for times when it's empty, if that's possible?

Reply:
I agree, however, this is consistent with the author archives. Should we default to the site description for both?

Reply:
I would actually opt to not include any `og:description` tag at all when we don't have any info. This way, Facebook's bot will go and crawl the page content to fill in that information on its own.

Reply:
There is nowhere in the function that sets up a default value for og:description and tags are only included if a value is set. 

With a category description:
<img width="440" alt="Screen Shot 2019-08-02 at 8 43 23 PM" src="https://user-images.githubusercontent.com/24902269/62404946-c6510180-b566-11e9-9d41-6c1f060c77c7.png">

Without a category description:
<img width="433" alt="Screen Shot 2019-08-02 at 8 43 50 PM" src="https://user-images.githubusercontent.com/24902269/62404948-d2d55a00-b566-11e9-8127-de0aa4d9b1e7.png">